stackslib/src/chainstate/burn/db/sortdb.rs

```diff
@@ -4502,6 +4490,21 @@ impl SortitionDB {
             .map(|(ch, bhh, _height)| (ch, bhh))
     }
 
+    #[cfg(test)]
+    pub fn set_canonical_stacks_chain_tip(
+        conn: &Connection,
+        ch: &ConsensusHash,
+        bhh: &BlockHeaderHash,
+        height: u64,
+    ) -> Result<(), db_error> {
+        let tip = SortitionDB::get_canonical_burn_chain_tip(conn)?;
+        let args: &[&dyn ToSql] = &[ch, bhh, &u64_to_sql(height)?, &tip.sortition_id];
+        conn.execute("UPDATE snapshots SET canonical_stacks_tip_consensus_hash = ?1, canonical_stacks_tip_hash = ?2, canonical_stacks_tip_height = ?3
+                    WHERE sortition_id = ?4", args)
+            .map_err(db_error::SqliteError)?;
+        Ok(())
+    }
```

Comment:
Like I commented elsewhere, functions that are test specific should, if possible, live in test modules.

Reply:
Sure